### ISSUE TITLE
Missing "globals" dependency

### ISSUE THREAD

**Issue by spalger** [eslint/eslintrc#1]
When I'm installing the latest version of eslint in my project, it pulls in this package which gets the wrong version of the `globals` package because it's not listed in the package.json for this dependency and `globals.es2015` is not defined by the version installed globalls in my project (as a side-effect of `babel-traverse`).

PR incoming

**Comment by LvChengbin** [comment]
Not only `globals` is missing.

```sh
(node:6385) [MODULE_NOT_FOUND] Error: @eslint/eslintrc tried to access globals, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6385) [MODULE_NOT_FOUND] Error: @eslint/eslintrc tried to access lodash, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.
(node:6385) [MODULE_NOT_FOUND] Error: debug tried to access supports-color (a peer dependency) but it isn't provided by its ancestors; this makes the require call ambiguous and unsound.
(node:6385) [MODULE_NOT_FOUND] Error: @eslint/eslintrc tried to access ignore, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.
(node:6385) [MODULE_NOT_FOUND] Error: @eslint/eslintrc tried to access minimatch, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.
```

**Comment by nzakas** [comment]
Fixed in #5.

**Comment by Rocamonde** [comment]
Sorry, just seen this as I had opened another issue for this.

With all the respect that I have for the ESLint project, it seems a bit sloppy to release a new ESLint version that does not even run a plain `yarn run eslint --version`. Is this an issue only on some machines, or how come this has not come up in testing?

If I can help in any way, please do let me know.
